### ISSUE TITLE
Contribute to ass 9 of project euler

### ISSUE THREAD

**Issue by paraggoyal28** [SouravBear8850/assignment#3]
I would like to contribute to this repository by adding ass 9 in java of project euler.

@Swastik2000 @Zeo-shark Please assign this task to me.

#hackoctoberfest2020

**Comment by Swastik2000** [comment]
Carry on 👍

**Comment by Zeo-shark** [comment]
@paraggoyal28 Yes Sure feel free to Contribute to this Repositories through PRs.
